### PR TITLE
Add WPForms Lite

### LINKED ISSUE #16: Add support for WPForms
Add support for WPForms: https://wordpress.org/plugins/wpforms-lite/

### PR DESCRIPTION
**Pull request by publicarray**
fixes #16

### CODE REVIEW

**Comment by e271828-** [comment]
@publicarray Thanks! Please update the readme list as well.

**Comment by publicarray** [comment]
@e271828- Thanks, It's Done!

File: wpforms/wpforms.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * Action that fires during form entry processing after initial field validation.
+ *
+ * @link   https://wpforms.com/developers/wpforms_process/
+ *
+ * @param  array  $fields    Sanitized entry field. values/properties.
+ * @param  array  $entry     Original $_POST global.
+ * @param  array  $form_data Form data and settings.
+ *
+ */
+function hcapture_wpforms_validate($fields, $entry, $form_data) {
```
Comment (e271828-) [inline wpforms/wpforms.php:37]:
```suggestion
function hcaptcha_wpforms_validate($fields, $entry, $form_data) {
```

File: wpforms/wpforms.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		'hcaptcha_wpforms_nounce',
+		'hcaptcha_wpforms'
+	);
+
+	if ( null === $error_message ) {
+		return $fields;
+	}
+	wpforms()->process->errors[ $form_data['id'] ] [ 'footer' ] = __( $error_message, 'hcaptcha-for-forms-and-more' );
+}
+add_filter( 'wpforms_process', 'hcapture_wpforms_validate', 10, 3 );
```
Comment (e271828-) [inline wpforms/wpforms.php:48]:
```suggestion
add_filter( 'wpforms_process', 'hcaptcha_wpforms_validate', 10, 3 );
```

**Comment by e271828-** [comment]
@publicarray would you mind `s/hcapture_wpforms_display/hcaptcha_wpforms_display/g`

**Comment by publicarray** [comment]
Oh oops, fixed it!

**Comment by sbconslt** [comment]
If this actually works a lot of users including me will be very grateful. Please fast track to release!

**Pull request merged**